src/components/common/Problem.vue

```diff
@@ -29,34 +30,107 @@ export default defineComponent({
 		const _file: Ref<string> = ref(props.file);
 		const renderDiv = ref<HTMLElement>();
 
+		// modified from https://github.com/tserkov/vue-plugin-load-script
+		async function loadScript(src: string) {
+			console.log('loading scripts...', src);
+			// eslint-disable-line no-param-reassign
```

Comment:
Note that `eslint-disable-line` only works at the end of the line and only affects the line it is on.

In any case, this is not needed.  Eslint does not warn about this.